### PR TITLE
refactor: Update electron-builder to version 20.13.3

### PR DESCRIPTION
**Pull request by ffflorian**
Builds tested on:
- [x] Linux
- [ ] macOS
- [ ] Windows

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -25,13 +25,18 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
...
-  "GenericName": "The most secure collaboration platform",
-  "Categories": "Network;InstantMessaging;Chat;VideoConference",
-  "Keywords": "chat;encrypt;e2e;messenger;videocall",
-  "StartupWMClass": "Wire"
+const LINUX_SETTINGS = {
+  afterInstall: 'bin/deb/after-install.tpl',
+  afterRemove: 'bin/deb/after-remove.tpl',
+  category: 'Network',
+  desktop: {
+    Version: '1.1',
```
Comment (lipis) [inline Gruntfile.js]: maybe use the `'<%= info.version %>'` here? Was that some sort of left over?

File: Gruntfile.js [hunk 2]
```diff
@@ -25,13 +25,18 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
...
+    Name: 'Wire',
```
Comment (lipis) [inline Gruntfile.js]: '<%= info.name %>'

File: Gruntfile.js [hunk 3]
```diff
@@ -25,13 +25,18 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
...
+    GenericName: 'The most secure collaboration platform',
+    Categories: 'Network;InstantMessaging;Chat;VideoConference',
+    Keywords: 'chat;encrypt;e2e;messenger;videocall',
+    StartupWMClass: 'Wire',
```
Comment (lipis) [inline Gruntfile.js]: `'<%= info.name %>'`

File: Gruntfile.js (hunk 1 above)
Comment (ffflorian) [reply Gruntfile.js]: Yeah, seems like.

File: Gruntfile.js [hunk 4]
```diff
@@ -25,13 +25,18 @@ const ELECTRON_PACKAGE_JSON = 'electron/package.json';
...
-  "Keywords": "chat;encrypt;e2e;messenger;videocall",
-  "StartupWMClass": "Wire"
+const LINUX_SETTINGS = {
+  afterInstall: 'bin/deb/after-install.tpl',
+  afterRemove: 'bin/deb/after-remove.tpl',
+  category: 'Network',
+  desktop: {
+    Version: '<%= info.version %>.<%= buildNumber %>',
+    Name: '<%= info.name %>',
+    GenericName: 'The most secure collaboration platform',
```
Comment (lipis) [inline Gruntfile.js]: `'<%= info.description %>'`, but we should possibly update the [actual description](https://github.com/wireapp/wire-desktop/blob/master/info.json#L3).

Comment (lipis) [reply Gruntfile.js]: /cc @infotexture

File: electron/package.json [hunk 5]
```diff
@@ -24,7 +24,7 @@
     "uuid": "3.2.1",
     "winston": "https://github.com/wireapp/winston.git#2.2.0-e"
   },
-  "description": "Secure messenger for everyone.",
+  "description": "The most secure collaboration platform.",
```
Comment (lipis) [inline electron/package.json]: maybe updating here as well: https://github.com/wireapp/wire-desktop/blob/master/info.json#L3?

**Comment by lipis** [comment]
#1436

**Pull request merged**